<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentData</class>
 <widget class="QWidget" name="AppointmentData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="appId"/>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="status">
     <item>
      <property name="text">
       <string>scheduled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>completed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cancelled</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="dateTime"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="doctorId"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
